test/mix/tasks/phx.gen.auth_test.exs

```diff
@@ -192,16 +192,16 @@ defmodule Mix.Tasks.Phx.Gen.AuthTest do
 
       assert_file("lib/my_app_web/components/layouts/root.html.heex", fn file ->
         assert file =~
-                 ~S|<.link href={~p"/users/settings"}>Settings</.link>|
+                 ~r|<.link.*href={~p"/users/settings"}.*>|s
```

Comment:
Updated these assertions to regexes to accommodate the multi-lining needed to satisfy the formatter. Explicit matches are still covered in the injector test.